core/ui/i18n/locales/pt.ts

```
@@ -10,6 +10,11 @@ export const pt = {
   onboarding_description_4:
     'O Vultisig é uma carteira sem frases-semente. Em vez disso, usa fragmentos fáceis de gerenciar. Cada dispositivo tem seu próprio fragmento exclusivo que precisa de backup separado.',
   moreInfo: 'Mais informações',
+  incorrect_passcode: 'Código incorreto',
+  passcodes_do_not_match: 'Os códigos não correspondem',
+  digit_input_success_validation: 'Verificação bem-sucedida',
```

Biome (1.9.4)

[error] This property value named passcodes_do_not_match is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named passcodes_do_not_match


(lint/suspicious/noDuplicateObjectKeys)